app/fields/school_name_field.rb

```diff
@@ -2,6 +2,10 @@
 
 class SchoolNameField < Administrate::Field::Base
   def to_s
-    data.try(:name) || data.try(:local_id)
+    return 'None' unless data
```

Comment:
:)  Calling `object.to_s` and having it make database queries is pretty surprising, but this is limited to this one admin sorting thing, and if this kind of thing isn't supported in administrate, shipping a workaround and moving on seems great. 👍

Reply:
Tried a less surprising way of doing this but it didn't fix sorting. I'm going to go with the "ship ugly/weird but working" approach for now to fix as much as possible as quickly as possible.

Reply:
Yeah sounds great! 👍